desktop-widgets/about.h

```diff
@@ -9,7 +9,7 @@ class SubsurfaceAbout : public QDialog {
 	Q_OBJECT
 
 public:
-	explicit SubsurfaceAbout(QWidget *parent = 0, Qt::WindowFlags f = 0);
+	explicit SubsurfaceAbout(QWidget *parent = 0, Qt::WindowFlags f = QFlag(0));
```

Comment:
Changes in this commit are fine for now. Ultimately, I would remove these default parameters. Because either:
- We want these dialogs be generated with different flags. Then we shouldn't provide default parameters.
- We want these dialogs be generated with the same flags. Then just hardcode them in the constructor.